beesdoo_purchase/models/purchase.py

```diff
@@ -0,0 +1,20 @@
+# coding: utf-8
+
+from openerp import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+
+    _inherit = "purchase.order"
+
+    @api.multi
+    def _default_supervisor_id(self):
+        for rec in self:
+            if not rec.supervisor_id:
+                return rec.create_uid
+            else:
+                return rec.supervisor_id
+
+    create_uid = fields.Many2one('res.users', 'Creator')
+    supervisor_id = fields.Many2one('res.users', 'Responsible',
+                                    default=lambda self: self.env.user)
```

Comment:
Tu as créé un fonction `_default_supervisor_id` qui n'est pas utilisée. Je pense que la bonne façon d'initialiser `supervisor_id` est plutôt `default=lambda self: self.env.user` comme tu l'as fait ici.

Par ailleurs, j'alignerais les champs pour rendre la lecture plus claire et agréable:

```python
    supervisor_id = fields.Many2one(
        comodel_name='res.users',
        string='Responsible',
        default=lambda self: self.env.user)
```

Reply:
Ouille, j'ai oublié de nettoyer avant de faire mon commit. En effet cette fonction ne doit plus être là.

Explicit is better than implicit. Je vais changer dans ce sens. :)